test/e2e/flowaggregator_test.go

```diff
@@ -102,17 +102,14 @@ const (
 )
 
 func TestFlowAggregator(t *testing.T) {
-	// TODO: remove this limitation after flow aggregator supports IPv6
-	skipIfIPv6Cluster(t)
-	skipIfNotIPv4Cluster(t)
-	data, err, isIPv6 := setupTestWithIPFIXCollector(t)
+	data, isIPv6, err := setupTestWithIPFIXCollector(t)
```

Comment:
do we have to skip the scenario if the cluster is dual stack here as we do not have the support yet?

Reply:
Thanks, addressed.